easybuild/easyconfigs/f/FRANz/FRANz-2.0.0-foss-2018a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'FRANz'
+version = '2.0.0'
+
+homepage = 'https://www.bioinf.uni-leipzig.de/Software/FRANz'
+description = """A fast and flexible parentage inference program for natural populations."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['http://download.sourceforge.net/franzpedigree/']
+sources = [SOURCE_TAR_GZ]
+
+checksums = [
+    '429cf4b47b91c28711b7a5d37f28c982ee13229ff28af714c39a81a77b9281e1',
+]
```

Comment:
@bedroge - normally the `checksums` field would go directly beneath `sources` i.e. without a blank line in-between.
The `--inject-checksums` command does it this way for you.